Corona-Warn-App/src/test/java/de/rki/coronawarnapp/appconfig/mapping/ExposureWindowRiskCalculationConfigMapperTest.kt

```diff
@@ -0,0 +1,90 @@
+package de.rki.coronawarnapp.appconfig.mapping
+
+import de.rki.coronawarnapp.server.protocols.internal.v2.AppConfigAndroid
+import de.rki.coronawarnapp.server.protocols.internal.v2.RiskCalculationParametersOuterClass
+import io.kotest.assertions.throwables.shouldThrowAny
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+import testhelpers.BaseTest
+
+class ExposureWindowRiskCalculationConfigMapperTest : BaseTest() {
+
+    private fun createInstance() = ExposureWindowRiskCalculationConfigMapper()
+    private val defaultRiskParams = RiskCalculationParametersOuterClass.RiskCalculationParameters.getDefaultInstance()
+    private val defaultKeysMapping = AppConfigAndroid.DiagnosisKeysDataMapping.getDefaultInstance()
+
+    private val transmissionRiskValueMapping =
+        RiskCalculationParametersOuterClass.TransmissionRiskValueMapping.newBuilder()
+            .setTransmissionRiskLevel(3)
+            .setTransmissionRiskValue(0.6)
+    private val riskParams = RiskCalculationParametersOuterClass.RiskCalculationParameters.newBuilder()
+        .addTransmissionRiskValueMapping(transmissionRiskValueMapping)
+
+    @Test
+    fun `throws when riskCalculationParameters are missing`() {
+        val rawConfig = AppConfigAndroid.ApplicationConfigurationAndroid.newBuilder()
+            .build()
+
+        rawConfig.run {
+            hasRiskCalculationParameters() shouldBe false
+            hasDiagnosisKeysDataMapping() shouldBe false
+        }
+
+        shouldThrowAny {
```

Comment:
Would prefer to be more specific here, otherwise even something like a `NotMockedException` could accidentally cause the test to pass.

Maybe something like 

```kotlin
shouldThrow<ApplicationConfigurationInvalidException> {
    // Blub
}
```

Reply:
If desired, to check for a specific abort condition, maybe something like:

```kotlin
val error = shouldThrow<ApplicationConfigurationInvalidException> {
    // Blub
}
error.msg shouldContain "Transmission Risk Value"
```

Reply:
👌 
adjusted